src/pip/_internal/locations/__init__.py

```diff
@@ -45,13 +45,13 @@ def _default_base(*, user: bool) -> str:
 def _warn_if_mismatch(old: pathlib.Path, new: pathlib.Path, *, key: str) -> bool:
     if old == new:
         return False
-    issue_url = "https://github.com/pypa/pip/issues/9617"
+    issue_url = "https://github.com/pypa/pip/issues/10148"
```

Comment:
Let's file a new issue, with the explicit goal of being for just this thing?

It would suck to have lots of comments here.